docker/s3/compose.yml

```diff
@@ -0,0 +1,94 @@
+version: '3.8'
+services:
+  zookeeper:
```

Comment:
Instead of building, I'd just use https://hub.docker.com/r/confluentinc/cp-zookeeper

Reply:
Or even run Kafka in KRaft mode

Reply:
Here's a complete example https://github.com/aiven/python-kafka-protocol/tree/main/container

Reply:
It's actually using the same build as kafka service, so no much waste. 
Also wanted to avoid downloading +1GB of images that come with cp-kafka.

Though, if you feel strongly, I'm happy to replace -- hopefully at some point we test only with kraft :)

Reply:
Nah, it's ok then